jscomp/bsb/bsb_package_specs.ml

```diff
@@ -83,15 +87,14 @@ and from_json_single suffix (x : Ext_json_types.t) : Bsb_spec_set.spec =
             | Some _ | None -> false
           in
           let suffix =
-            match map.?("suffix") with
-            | Some (Str { str = suffix; loc }) ->
-                let s = Ext_js_suffix.of_string suffix in
-                if s = Unknown_extension then
-                  Bsb_exception.errorf ~loc "expected .js, .mjs, .cjs or .bs.js, .bs.mjs, .bs.cjs"
-                else s
+            match map.?(Bsb_build_schemas.suffix) with
+            | Some (Str { str = suffix; _ }) when validate_js_suffix suffix -> suffix
+            | Some (Str {str; loc}) ->
+              Bsb_exception.errorf ~loc
+                ("invalid js file extension \"%s\"") str
```

Comment:
This is user facing, right? Can we add the rules for valid extensions here?

Reply:
Yes, will update the PR.

Reply:
@zth Updated, please re-check.

Reply:
Perfect! 👍